CHANGELOG.md

```diff
@@ -12,11 +12,9 @@
 * [ENHANCEMENT] Expose `/sync/mutex/wait/total:seconds` Go runtime metric as `go_sync_mutex_wait_total_seconds_total` from all components. #5879
 * [ENHANCEMENT] Query-scheduler: improve latency with many concurrent queriers. #5880
 * [ENHANCEMENT] Implement support for `limit`, `limit_per_metric` and `metric` parameters for `<Prometheus HTTP prefix>/api/v1/metadata` endpoint. #5890
-* [ENHANCEMENT] Go: updated to 1.21.1. #5955
```

Comment:
Why was this removed?

Reply:
@pstibrany It was a duplicate: it is already present in CHANGELOG [here](https://github.com/grafana/mimir/blob/main/CHANGELOG.md?plain=1#L152). The reason: between creation of this PR and its merging, another release of mimir has happened.